trajectory_converter:
  ros__parameters:
    input_topic: "/planner/trajectory"
    output_topic: "/mpc/trajectory"

command_converter:
  ros__parameters:
    rates_input_topic: "/mpc/out/vehicle_rates_setpoint"
    motors_input_topic: "/mpc/out/actuator_motors"
    output_topic: "/controller/cmd"
